profiles/default/linux/alpha/17.0/desktop/gnome/systemd/deprecated

```diff
@@ -0,0 +1,32 @@
+default/linux/alpha/17.0/desktop/gnome/systemd/merged-usr
```

Comment:
I would add "Please see the ... news item for more information."

Reply:
Do you mean remove the news item text from the deprecated file and refer to it instead?

I don't see much value in linking off to pretty much exactly the same text.

Reply:
I see. I can mention the news item.